assets/dev/js/editor/regions/panel/pages/menu/views/group.js

```diff
@@ -17,7 +17,7 @@ module.exports = Marionette.CompositeView.extend( {
 		const callback = childView.model.get( 'callback' );
 
 		if ( _.isFunction( callback ) ) {
-			callback.call( childView );
+			callback.call( childView, childView );
```

Comment:
Maybe you don't need the second argument and use the `this` inside the callback?